bin/anthology/utils.py

```diff
@@ -82,17 +82,17 @@ def deconstruct_anthology_id(anthology_id):
         P18-1 -> ('P18', '1', None)
         W18-63 -> ('W18', '63', None)
     """
-    tokens = anthology_id.split('-')
+    collection_id, rest = anthology_id.split('-')
     if anthology_id.startswith('W') or anthology_id.startswith('C69'):
-        if len(tokens) == 3:
-            return (tokens[0], str(int(tokens[1])), str(int(tokens[2])))
+        if len(rest) == 4:
+            return (collection_id, str(int(rest[0:2])), str(int(rest[2:])))
         else:                   # Possible Volume only identifier
-            return (tokens[0], str(int(tokens[1])), None)
+            return (collection_id, str(int(rest)), None)
     else:
-        if len(tokens) == 3:
-            return (tokens[0], str(int(tokens[1])), str(int(tokens[2])))
+        if len(rest) == 4:
+            return (collection_id, str(int(rest[0:1])), str(int(rest[1:])))
         else:                   # Possible Volume only identifier
-            return (tokens[0], str(int(tokens[1])), None)
+            return (collection_id, str(int(rest)), None)
 
```

Comment:
This function is supposed to return (collection_id, volume_number, paper_number) from IDs like `P19-1001` or `P19-1`, but was not doing this at all. This fixes it.